### ISSUE TITLE
Not finding highest count

### ISSUE THREAD

**Issue by crazykarl7** [contactmayankp/AutoNumberUpdater#5]
Hello,
 I really like your product. I am using the current version and it seems to not be finding the highest value. It would appear that the number portion of the autonumber is being treated as a string instead of a number. For example we have OPP-99999. The tool is showing the next number to be 1000 and doesn't increment.  I have tried to set a seed number of 300000 and even with the current version of the tool it still showing OPP-1000 as the next available number.

**Comment by contactmayankp** [comment]
HI @crazykarl7 

are you sure you are using latest version of tool?  let me double check this one..

**Comment by crazykarl7** [comment]
It looks like I have a higher version than the latest version.
![image](https://user-images.githubusercontent.com/107001245/173363201-534e10b9-c005-4cd2-8a85-d95ca24bc739.png)

**Comment by contactmayankp** [comment]
Hi @crazykarl7 

Logic been used is to find latest record and increment number, if no record is present then we start with seed number...

how many records are present for your auto number entity in your environment?  is it possible to share your auto number config including seed you are using for this..

apologies but just want to know full details  and re-create this issue in my environment before applying fix for this so it can work in all scenario...

**Comment by crazykarl7** [comment]
Here is a screenshot of the field configuration. 
![image](https://user-images.githubusercontent.com/107001245/173583660-fd05c536-d09e-445f-83b6-7e99b5c57a05.png)

Attached is an excel of all of the records with an ID.  The tool shows the next value as OPP-1000. If I try to use it, it will just set records to OPP-1000 (hence why so many have that value).  There are about 65k records that need to be backfilled.
[IDs.xlsx](https://github.com/contactmayankp/AutoNumberUpdater/files/8899948/IDs.xlsx)

**Comment by contactmayankp** [comment]
Hi @crazykarl7 ,

I did try to re-produce this at my end and I have used following configuration for this..

[Note I mentioned digit as 6 because your seed value is 300000 and to match you need these many digits..

![image](https://user-images.githubusercontent.com/6472981/173812060-311b88fd-117a-4f3c-a405-6fe8e5a940ff.png)

I am using following version of tool..

![image](https://user-images.githubusercontent.com/6472981/173812306-4202ab06-ba67-482f-a2d3-caf3afab2867.png)



Here is run output for this field.... inside tool and dynamics...

![image](https://user-images.githubusercontent.com/6472981/173812527-fc04bbb7-4b46-4bc9-a048-deb184fc3029.png)

![image](https://user-images.githubusercontent.com/6472981/173812636-da800159-45c2-47f9-bdc4-0e24feec8c4e.png)


I also create new record in dynamics to see if new values gets assigned or not..

this also applied new value correctly...
![image](https://user-images.githubusercontent.com/6472981/173813243-6a541285-0ee5-4920-961a-3da854ce1ecf.png)


can you please change minimum digit field and also blank out few duplicate values on this field (so this tool only updates this autonumber field when it is blank (i.e. NULL)) and see if this works on your end or not.....

**Comment by crazykarl7** [comment]
I tried changing to 6. When I tried the tool it threw an error:
![image](https://user-images.githubusercontent.com/107001245/174126035-563f50ff-2be3-4a3a-aec3-4fbe76d7de9e.png)

Here is the exception text:
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.Exception: That was hard. Couldn't even guess what current SEQNUM is.
Numbered value for last created record is:  
OPP-99999
   at Sdmsols.XTB.AutoNumberUpdater.AutoNumberUpdater.GuessSeed()
   at Sdmsols.XTB.AutoNumberUpdater.AutoNumberUpdater.cmbAttributes_SelectedIndexChanged(Object sender, EventArgs e)
   at System.Windows.Forms.ComboBox.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ComboBox.WmReflectCommand(Message& m)
   at System.Windows.Forms.ComboBox.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
